PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/utils/tlp.cpp

```diff
@@ -15,6 +17,31 @@ boost::dynamic_bitset<> TLP::getBitRep() {
 	return result;
 }
 
+
+/**
+ * int headerConsumption;
+	int dataConsumption;
+	TLPHeader* header;
+	Dllp::CreditType creditConsumedType;
+	boost::dynamic_bitset<> dataPayload;
+*/
+TLP* TLP::getObjRep(boost::dynamic_bitset<> bitset) {
+	TLP* tlp = new TLP();
+	int size = bitset.size();
+	boost::dynamic_bitset<> length_bitset = get_bits(bitset, size - 32, size - 23);
+	int lengthValue = length_bitset.to_ulong();
+
+	boost::dynamic_bitset<> TLPType_bitset = get_bits(bitset, size - 8, size - 1);
+	if (TLPType_bitset.to_ulong() == 64 ||96 ) { //64 -->MemWrite32 96-->MemWrite64
```

Comment:
Configuration writes and completions with data also have data payloads, so they need to be added